bitnami/wordpress/README.md

```diff
@@ -99,6 +97,8 @@ The following table lists the configurable parameters of the WordPress chart and
 | `wordpressConfiguration`               | The content for your custom wp-config.php file (experimental feature)                     | `nil`              |
 | `existingWordPressConfigurationSecret` | The name of an existing secret with your custom wp-config.php file (experimental feature) | `nil`              |
 | `wordpressConfigureCache`              | Enable W3 Total Cache plugin and configure cache settings                                 | `false`            |
+| `apacheConfiguration`                  | The content for your custom httpd.conf file (experimental feature)                        | `nil`              |
```

Comment:
I am not very sure of the "experimental feature" note.
Maybe we should reword it.